js/reducers/createReducer.js

```diff
@@ -0,0 +1,39 @@
+const initiellState = { data: {} };
+
+export const createReducer = (feilActionType,
+    henterActionType,
+    hentetActionType,
+    initState = initiellState,
+    mapper = null) => {
+    return (state = initState, action = {}) => {
+        switch (action.type) {
+            case feilActionType: {
+                return {
+                    ...state,
+                    henter: false,
```

Comment:
Er dette en mulighet til å snike in `status` med en enum i stedet for bool felt for alle statusene? 😇

Reply:
@MartinMekk God idé det, men jeg kan ta det som en egen PR siden det medfører endring over alt der bool-feltet brukes.